test/Driver/driver-time-compilation.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %swiftc_driver -driver-time-compilation %s 2>&1 | %FileCheck %s
+
+// CHECK: Driver Time Compilation
+// CHECK: Total Execution Time: {{[0-9]+}}.{{[0-9]+}} seconds ({{[0-9]+}}.{{[0-9]+}} wall clock)
+// CHECK: ---Wall Time---
+// CHECK: --- Name ---
+// CHECK: {{[0-9]+}}.{{[0-9]+}} (100.0%)  Total
```

Comment:
I think it's worth checking that the name of the file appears in the output too.